src/content/en/updates/2017/07/chrome-61-media-updates.md

```diff
@@ -0,0 +1,92 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the media updates in Chrome 61.
+
+{# wf_updated_on: 2017-07-17 #}
+{# wf_published_on: 2017-07-17 #}
+{# wf_tags: news,chrome61,media #}
+{# wf_featured_image: /web/updates/images/generic/animations.png #}
+{# wf_featured_snippet: Background video track optimization and automatic video fullscreen when device is rotating features are there! #}
+
+# Media Updates in Chrome 61 {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+- Chrome now [disables video tracks when a MSE video is played in the
+  background](#background-video-track-optimization) to optimize performance. 
+- A video will [automatically go fullscreen when device is
+  rotating](#auto-fullscreen-rotate).
+
+## Background video track optimization (MSE only) {: #background-video-track-optimization}
+
+To improve battery life, Chrome now disables video tracks when the video is
+played in the background if this one uses [Media Source Extensions (MSE)].
+
+You can inspect these changes by going to the `chrome://media-internals` page,
+and filter for the "info" property. When the tab containing a playing video
+becomes inactive, you'll see a message like `Selected video track: []`
+indicating that the video track has been disabled. When tab becomes active
+again, video track is re-enabled automatically.
+
+<figure>
+  <img src="/web/updates/images/2017/07/media-internals.png"
+       alt="Log panel in the chrome://media-internals page">
+  <figcaption>
+    <b>Figure 1.</b>
+    Log panel in the <i>chrome://media-internals</i> page
+  </figcaption>
+</figure>
+
+For those who want to understand what is happening, here's some JavaScript code
+snippet that shows you what Chrome is roughly doing behind the scenes.
+
+    var video = document.querySelector('video');
+    var selectedVideoTrackIndex;
+
+    document.addEventListener('visibilitychange', function() {
+      if (document.hidden) {
+        // Disable video track when page is hidden.
+        selectedVideoTrackIndex = video.videoTracks.selectedIndex;
+        video.videoTracks[selectedVideoTrackIndex].selected = false;
+      } else {
+        // Re-enable video track when page is not hidden anymore.
+        video.videoTracks[selectedVideoTrackIndex].selected = true;
+      }
+    });
```

Comment:
Should we suggest that developers reduce the quality of the streem if they think it's worth it?